FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -88,26 +95,25 @@
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;DistressedRatingsDowngrade">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;Downgrade"/>
 		<rdfs:label xml:lang="en">distressed ratings downgrade</rdfs:label>
-		<skos:definition xml:lang="en">credit event triggered when the credit rating of the reference obligation is downgraded to a distressed rating level or the rating is withdrawn by one or more ratings agency</skos:definition>
+		<skos:definition xml:lang="en">credit event triggered when the credit rating of a party or reference obligation is downgraded to a distressed rating level or the rating is withdrawn by one or more ratings agency</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;Downgrade">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;ObligationSpecificCreditEvent"/>
 		<rdfs:label xml:lang="en">downgrade</rdfs:label>
-		<skos:definition xml:lang="en">credit event triggered when the credit rating of the reference obligation is downgraded</skos:definition>
+		<skos:definition xml:lang="en">credit event triggered when the credit rating of a party or reference obligation is lowered</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;EntitySpecificCreditEvent">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;CreditEvent"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;involves"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-org-fm;FormalOrganization"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-le-lp;LegalPerson"/>
```

Comment:
"involves" seems too vague here if it's intended to represent the entity subject to the event. For example a Downgrade also "involves" the credit rating agency.

Reply:
changed involves to appliesTo